pkg/varlogtest/admin.go

```diff
@@ -294,8 +294,7 @@ func (c *testAdmin) AddLogStream(_ context.Context, topicID types.TopicID, logSt
 
 	c.vt.logStreams[logStreamID] = lsd
 
-	invalidLogEntry := varlogpb.InvalidLogEntry()
-	c.vt.localLogEntries[logStreamID] = []*varlogpb.LogEntry{&invalidLogEntry}
+	c.vt.localLogEntries[logStreamID] = []*varlogpb.LogEntry{{}}
```

Comment:
What does `{{}}` syntax mean? 
Can we replace it with `c.vt.localLogEntries[logStreamID] = nil`?

Reply:
No, we can't. The `c.vt.localLogEntries[logStreamID]` is a slice with a single entry, an invalid log entry. `{{}}` is a slice with zero value, which is an invalid log entry.